### PR TITLE
Add Ripe

### PR DESCRIPTION
**Pull request by radhta**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -7386,6 +7386,14 @@
...
     },
 
+    {
+        "name": "Ripe NCC",
+        "difficulty": "impossible",
+        "domains": [
+            "ripe.net"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:7396]:
```suggestion

```

File: _data/sites.json [hunk 2]
```diff
@@ -7386,6 +7386,14 @@
         ]
     },
 
+    {
+        "name": "Ripe NCC",
+        "difficulty": "impossible",
```
Comment (tupaschoal) [inline _data/sites.json:7391]: Can you please provide the url, preferably one that endorses that it's not possible to delete an account?

Comment (radhta) [reply _data/sites.json:7391]: There's no mention of account deletion anywhere to be found, not even the privacy policy.

Comment (radhta) [reply _data/sites.json:7391]:
```suggestion
        "name": "Ripe NCC",
        "difficulty": "impossible",
        "url": "",
```
might be feasible.

Comment (tupaschoal) [reply _data/sites.json:7391]: If there's no privacy policy or faq link, I'd rather you just added the plain url to the site. All of our links are clickable, so we require such.

Comment (radhta) [reply _data/sites.json]: Changes pushed, and commits merged.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**